js/interface/add-markers.js

```diff
@@ -320,16 +320,29 @@ Fliplet.InteractiveMap.component('add-markers', {
       this.allMarkerStyles.push(newItem)
       this.saveData()
     },
-    deleteMarkerStyle(index) {
+    deleteMarkerStyle(index, marker) {
       Fliplet.Modal.confirm({
         title: 'Delete marker style',
         message: '<p>You will have to manually update any marker that has this style applied.</p><p>Are you sure you want to delete this marker style?</p>'
-      }).then((result) => {
```

Comment:
Any reason why the brackets were removed? We use brackets everywhere else.

Reply:
I know this is correct syntax, but removing here and not removing elsewhere is creating inconsistency.

Reply:
@hcarneiro 
I already removed a lot of these brackets in old PRs

Reply:
That's OK then, I wasn't aware of that.